### PR TITLE
website: testing content checks

### PR DESCRIPTION
**Pull request by BRKalow**

### CODE REVIEW

File: website/data/docs-nav-data.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
     "title": "What is Waypoint?",
     "routes": [
       {
-        "title": "Overview",
         "path": "intro"
```
Comment (BRKalow) [inline website/data/docs-nav-data.json:6]:
```suggestion
				"title": "Overview",
        "path": "intro"
```

File: website/data/docs-nav-data.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "title": "What is Waypoint?",
     "routes": [
       {
-        "title": "Overview",
+				"title": "Overview",
```
Comment (BRKalow) [inline website/data/docs-nav-data.json:6]:
```suggestion
```

**Comment by briancain** [comment]
Can this PR be closed @BRKalow ?

**Comment by BRKalow** [comment]
Yeah, thanks for the nudge!

**Pull request closed without merging**